src/WebJobs.Script.WebHost/Filters/SystemLevelAuthorizationAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Filters
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, Inherited = true, AllowMultiple = true)]
+    public sealed class SystemLevelAuthorizationAttribute : AuthorizationLevelAttribute
```

Comment:
nit: consider making naming match the pattern of the base class. E.g. SystemAuthorizationLevelAttribute.